src/platform/packages/shared/shared-ux/error_boundary/src/ui/error_boundary.tsx

```diff
@@ -62,18 +58,11 @@ class ErrorBoundaryInternal extends React.Component<
             error={error}
             errorInfo={errorInfo}
             name={componentName}
-            onClickRefresh={this.props.services.onClickRefresh}
+            onRecoverAttempt={this.props.services.onClickRefresh}
```

Comment:
Why are we renaming this prop? The intention for "recovering" is to refresh the page, which is also how the button is labeled. There do not need to be other methods to "attempt to recover."

Reply:
Isn't `The intention for "recovering" is to refresh the page` an implementation detail? SPA may re-render some parts of the app and refetch necessary lazy modules and data. While it's much harder than reloading a page it's still possible.

What if a consumer redefines `onClickRefresh`? According to `KibanaErrorBoundaryProvider` implementation it's possible to do that at the top level.